code/modules/mob/death.dm

```diff
@@ -55,6 +55,11 @@
 			spell_master.on_holder_death(src)
 	if(transmogged_from)
 		transmog_death()
+	if(client || mind)
+		for(var/mob/M in get_deadchat_hearers())
+			var/rendered = "\proper<a href='?src=\ref[M];follow2=\ref[M];follow=\ref[src]'>(Follow)</a><span class='game deadsay'> \The <span class='name'>[src]</span> has died at \the <span class='name'>[get_area(src)]</span>.</span>"
+			to_chat(M, rendered)
+			log_game("[key_name(src)], played by has died at [get_area(src)]. Coordinates: ([get_coordinates_string(src)])")
```

Comment:
>[ckey], played by has died at primary tool storage

played by whom???

Reply:
Oops, I forgot to remove the "played by" bit. `key_name(src)`, meanwhile, tells you both a player's ckey and character name.
I'll fix this now.